test/test-utils.js

```diff
@@ -168,7 +170,8 @@ export function setup(GameClass, hands, test_options = {}) {
 	testShortForms = shortForms.toSpliced(0, 0, 'x');
 
 	const state = new State(playerNames, 0, variant, {});
-	const game = new GameClass(-1, state, false, test_options.level ?? 1);
+	const [minLevel, maxLevel] = [test_options?.level?.min || 1, test_options?.level?.max || MAX_H_LEVEL];
```

Comment:
It doesn't matter too much here since 0 is an invalid level to provide, but when dealing with `undefined` values (such as those from optional chaining), it's often more correct to use the nullish coalescing operator `??` rather than the logical OR operator `||`. This is because all falsy values including `''`, `[]` and `0`, will trigger the logical OR operator, whereas only `null` and `undefined` will trigger the nullish coalescing operator.

Reply:
Thanks, bad habits from before that operator was available :-)